src/myrtlespeech/configs/deep_speech_2_en.config

```diff
@@ -13,6 +13,7 @@ speech_to_text {
   pre_process_step {
     stage: TRAIN_AND_EVAL;
     standardize {
+      norm_type: ALL_FEATURES;
```

Comment:
i.e so that behaviour doesn't change although it's likely that setting this to PER_FEATURE will be better.